libbeat/metric/system/memory/memory.go

```diff
@@ -1,3 +1,5 @@
+// +build darwin freebsd linux openbsd windows
```

Comment:
@kvch I readded these build tags as I'm not sure why they were removed in the previous PR? The part I didn't test is if the flags are still all correct as there are is potentially new functionality. Was there a reason you removed these?

Reply:
I think I removed it, because at some point netbsd and solaris did not compile. As I see, now it works. It might got lost in the ocean of patches I produced. Thanks for adding it again.